localstack/testing/aws/lambda_utils.py

```diff
@@ -35,21 +37,18 @@
     **dict.fromkeys(RUNTIMES_AGGREGATED.get("nodejs"), "index.handler"),
     **dict.fromkeys(RUNTIMES_AGGREGATED.get("ruby"), "function.handler"),
     **dict.fromkeys(RUNTIMES_AGGREGATED.get("java"), "echo.Handler"),
-    **dict.fromkeys(RUNTIMES_AGGREGATED.get("custom"), "function.handler"),
-    **dict.fromkeys(RUNTIMES_AGGREGATED.get("go"), "main"),
-    "dotnetcore3.1": "dotnetcore31::dotnetcore31.Function::FunctionHandler",  # TODO lets see if we can accumulate those
-    "dotnet6": "dotnet6::dotnet6.Function::FunctionHandler",
+    # TODO: check if "function.handler" is relevant; or just main or bootstrap?!
```

Comment:
It generally is not, unless the custom runtime reads it in some way. However, some value has to be put in there :)

Reply:
Good clarification 👍  replaced TODO with a comment